magento/magento2
# FIXED: Discount fixed amount whole cart applied mutiple time when customer use Check Out with Multiple Addresses

Fixed Issue with CartFixed and Multishipping.
### Description (*)

I have fixed issue #25834 in this PR
### Fixed Issues (if relevant)

1. #25834 

### Manual testing scenarios (*)

1. Create cart rule, no condition, action: Fixed amount discount for whole cart. Discount amount $10
2. Create account, create 2 addresses: address 1, address 2
3. Add 2 SKUs to cart: SKU1, SKU2
4. Check Out with Multiple Addresses
5. SKU1 ship to address 1, SKU2 ship to address 2
6. Order review page discount applied once. https://prnt.sc/qoju2d 
7. Placed an order discount applied in single order. 

### Questions or comments


### Contribution checklist (*)
